crates/symbolicator-sources/src/remotefile.rs

```diff
@@ -88,6 +88,8 @@ pub struct SourceLocationIter<'a> {
     /// Filters from a `SourceConfig` to limit the amount of generated paths.
     filters: &'a SourceFilters,
 
+    index: Option<&'a SymstoreIndex>,
```

Comment:
Maybe this should be combined into `SourceFilters`?

Or at least more generic, this seems awfully wrong to reference a specific index implementation here.